### ISSUE TITLE
Unable to find exact match

### ISSUE THREAD

**Issue by phptek** [Firesphere/silverstripe-solr-search#167]
**Describe the bug**

I would like to be able to perform a search for a page with a known title, and have that page returned in results, which currently doesn't happen for me.

**To Reproduce**

1. Create a page with a Title of "Random Page Title"
2. Create a bunch of unrelated pages with different values for their Title field
3. Run Configure + Index tasks
4. Search for: `Random Page Title` (No quotes)
5. Observed: N results. But the desired page is not returned. Other roughly matching results are.
6. Search for: `"Random Page Title"` (With quotes)
7. Observed: 0 results. The desired page is not returned, nor are any others

I can search in the Solr UI directly, and search for `"Random Page Title"` (With quotes) and the desired page is returned as the _only_ result.

**Expected behavior**

Any one of the following:

1. I can search for `"Random Page Title"` (With quotes) and the desired page is returned as the _only_ result.
2. I can search for `Random Page Title` (Without quotes) and the desired page is returned as the _only_ result.
3. I can search for `"Random Page Title"` (With quotes) and the desired page is returned as the _first_ result (Alongside other related results).
2. I can search for `Random Page Title` (Without quotes) and the desired page is returned as the _first_ result (Alongside other related results).

**Environment**

PHP 7.2 + Bionic (Vagrant / LXD) + Module `2974936ea` + Silverstripe 4.4.3

Search Index:

```
Catalyst\TRC\Search\TRCIndex:
  MyIndex:
    Classes:
      - SilverStripe\CMS\Model\SiteTree
      - SilverStripe\Assets\File
      - DNADesign\Elemental\Models\BaseElement
    FulltextFields:
      - Title
      - forTemplate
      - LeftColumn
      - RightColumn
      - Content
      - Intro
    FilterFields:
      - SubsiteID
      - ShowInSearch
      - ClassName
```

Query Construction:

```
$params = [[
    'text' => $keywords, // <-- e.g. Random Page Title OR "Random Page Title"
    'fuzzy' => false,
    'boost' => false,
]];

$query = SearchQuery::create()
        ->setClasses(self::$classes_to_search)
        ->setStart($start)
        ->setFields($params[0])
        ->setTerms($params)
        ->setRows(100)
        ->setExclude($exclusions);
```

**Comment by Firesphere** [comment]
Possibly a XML error somewhere. Needs investigation

**Comment by Firesphere** [comment]
Could you please supply a Debug trace of the query executed? I can't seem to reproduce this.

**Comment by Firesphere** [comment]
Should be resolved in #171 , closing the issue for now to be discussed in the PR.
